app/src/lang/translations/en-US.yaml

```diff
@@ -418,6 +418,9 @@ delete_collection: Delete Collection
 update_collection_success: Updated Collection
 delete_collection_success: Deleted Collection
 start_end_of_count_items: '{start}-{end} of {count} items'
+start_end_of_count_filtered_items: '{start}-{end} of {count} filtered items'
+start_item_singular: '{start} item'
```

Comment:
To extend that same thought, instead of passing a hardcoded `1` to this translation, we could just make it say `One Item` 🙂

Reply:
Hmm, not sure. I wouldn't really like seeing 'Eén item' in Dutch for example. I'd rather see '1 item'.